docs/api.md

```diff
@@ -1848,6 +1881,157 @@ This endpoint supports retrieving extended metadata; provide a header `X-Extende
+    + Attributes (Error 400)
+
++ Response 403 (application/json)
+    + Attributes (Error 403)
+
++ Response 409 (application/json)
+    + Attributes (Error 409)
+
+### Updating Submission Data [PUT /v1/projects/{projectId}/forms/{xmlFormId}/submissions/{instanceId}]
+
+_(introduced: version 1.2)_
+
+You can use this endpoint to submit _updates_ to an existing submission.
+
+The `instanceId` that is submitted with the initial version of the submission is used permanently to reference that submission logically, which is to say the initial submission and all its subsequent versions. Each subsequent version will also provide its own `instanceId`. This `instanceId` becomes that particular version's identifier.
+
+To perform an update, you need to provide in the submission XML an additional [`deprecatedID` metadata node](https://getodk.github.io/xforms-spec/#metadata) with the `instanceID` of the particular submission version you are replacing. If the `deprecatedID` you give is anything other than the identifier of the current version of the submission at the time the server receives it, you will get a `409 Conflict` back.
+
+The XML data you send will _replace_ the existing data entirely. All of the data must be present in the updated XML.
+
+When you create a new submission version, any uploaded media files attached to the current version that match expected attachment names in the new version will automatically be copied over to the new version. So if you don't make any changes to media files, there is no need to resubmit them.
+
++ Parameters
+    + xmlFormId: `simple` (string, required) - The `xmlFormId` of the Form being referenced.
+    + instanceId: `uuid:85cb9aff-005e-4edd-9739-dc9c1a829c44` (string, required) - The `instanceId` of the Submission being updated.
+
++ Request (application/xml)
+    + Body
+
+            <data id="simple">
+              <orx:meta>
+                <orx:deprecatedID>uuid:315c2f74-c8fc-4606-ae3f-22f8983e441e</orx:deprecatedID>
+                <orx:instanceID>uuid:85cb9aff-005e-4edd-9739-dc9c1a829c44</orx:instanceID>
+              </orx:meta>
+              <name>Alice</name>
+              <age>36</age>
+            </data>
+
++ Response 200 (application/json)
+    + Attributes (Submission)
+
++ Response 400 (application/json)
+    + Attributes (Error 400)
+
++ Response 403 (application/json)
+    + Attributes (Error 403)
+
++ Response 409 (application/json)
+    + Attributes (Error 409)
+
+### Getting an Enketo Edit URL [GET /v1/projects/{projectId}/forms/{xmlFormId}/submissions/{instanceId}/edit]
+
+_(introduced: version 1.2)_
+
+This endpoint redirects the user to an Enketo-powered page that allows the user to interactively edit th e submission. Once the user is satisfied, they can perform the update submission directly through the Enketo interface.
+
+The Enketo instance is already hosted inside of ODK Central. There is no reason to create or use a separate Enketo installation.
```

Comment:
(For production setups / the full Central setup...)

> There is no reason to create or use a separate Enketo installation.

You might want to set up a separate enketo installation for dev work.